### PR TITLE
Initialize frontend Django app

### LINKED ISSUE #3: Create Django app for UI
`./manage.py startapp registrationui`

The app should contain a page for associating tickets with armbands, and pulling in information from the ticketing system.

flowgraph

Additionally (but not needed for the very first iteration, just keep it in mind):
- it should be possible to register crew, who may not have a ticket in the system
- we need a way to register day-passes, which do *not* have a normal ticket in the system

### PR DESCRIPTION
**Pull request by WindShiba**
il delete branch, and make another after this. Also feel free to flame me.

Closes #3

### CODE REVIEW

File: registration/urls.py [hunk 1]
```diff
@@ -14,10 +14,17 @@
...
+#from frontend.views import base_view, test_view
+
+
+#For now we're using function based views, but we might change to class based views later on if needed. We might need it for registration api users.?
 
 urlpatterns = [
-    path("admin/", admin.site.urls),
-]
+    path('admin/', admin.site.urls), # This is the admin page for the project.
+    path('', TemplateView.as_view(template_name='base.html'), name='home'),  # Root URL -> base.html
```
Comment (oddstr13) [inline registration/urls.py]: I don't think we should be using the TemplateView here, I'd rather have a function view with the render macro.
Easier to pass database data to the template at a later point then.

File: registration/settings.py [hunk 2]
```diff
@@ -20,10 +25,10 @@
 # See https://docs.djangoproject.com/en/5.1/howto/deployment/checklist/
 
 # SECURITY WARNING: keep the secret key used in production secret!
-SECRET_KEY = "django-insecure-5*-kc^mwgvy)2g(8@lj$q6=ulwbj2b7!^8vyk28nvgaqmmmsp5"
+SECRET_KEY = os.getenv('DJANGO_SECRET_KEY')
 
 # SECURITY WARNING: don't run with debug turned on in production!
-DEBUG = True
+DEBUG = os.getenv('DJANGO_DEBUG')
```
Comment (oddstr13) [inline registration/settings.py]: A default value would be a good idea here I think, defaulting to the production setting (debug off) should be fine.
```suggestion
DEBUG = os.getenv('DJANGO_DEBUG', False)
```

File: frontend/views.py [hunk 3]
```diff
@@ -0,0 +1,10 @@
+from django.shortcuts import render
+from django.shortcuts import render, redirect
+from django.http import HttpResponse, HttpResponseNotFound
+
+
+# Create your views here. Class based views, we scrap the idea for now and go for function based views.
+""" def base_view(request):
```
Comment (oddstr13) [inline frontend/views.py]: Uncomment and use these instead of the TemplateView you're currently using

**Review by oddstr13 — CHANGES_REQUESTED** [review]
Not too important (yet), but you're missing the final newline in several of the files here (you can tell by the stop sign at the bottom of the diffs here on GitHub)

**Comment by oddstr13** [comment]
Change title to present tense; Add …

File: frontend/views.py (hunk 3 above)
Comment (WindShiba) [reply frontend/views.py]: sounds good, me fix

File: registration/urls.py (hunk 1 above)
Comment (WindShiba) [reply registration/urls.py]: That makes sense. I will switch to creating a function that renders a template in views.py instead.

**Review by oddstr13 — APPROVED** [review]

**Pull request merged**